client/src/components/UserInfo/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import './style.css';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import LoadingSpinner from '../common/LoadingSpinner';
+
+const UserInfo = () => {
+  const { user: { id, role } } = useSelector((state) => state.auth);
+  const [user, setUser] = useState({});
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    if (id) {
+      axios.get(`/api/user/info/${id}`).then((res) => {
+        setUser(res.data.data);
+        setLoading(false);
+      });
+    }
+  }, [id]);
+
+  const { name, bio, profilePicture } = user;
+
+  return (
+    <div className="user__info-section">
+      {loading ? (
+        <LoadingSpinner />
+      ) : (
+        user && (
+          <>
+            <div className="user__primary">
+              <div className="user__image-container">
+                <img src={profilePicture} alt="" />
+              </div>
+              <div className="user__primary-info">
+                <p className="user__name">{name}</p>
+                {role === 'interviewee' && (
+                  <button type="button" className="user__start-interview">
+                    Start a Practice Interview
+                  </button>
```

Comment:
This will not be shown to all interviewees, just the one who is already in his/her profile.